vuepress/docs/v7.0/tutorials/solution/entando-hub.md

```diff
@@ -114,12 +114,16 @@ Notes:
 Clicking the `Add +` button at the top of the page displays the form for creating a new Bundle Group:
 
 ![hub-add.png](./hub-images/hub-add.png)
+
+There are two ways to provide access to a Bundle Group:
+- Enter the URL for the Bundle Group source code under `Add URL Bundle`. 
```

Comment:
Change to "Add one (or more) bundles using `Add URL Bundle".  Side note, I felt the label was backwards here so in the next version it will be "Add Bundle URL" instead of "Add URL Bundle". Side note#2, the bundle source (distinct from the bundle URL used to deploy the bundle) is a separate field which will be added in Hub 2.1.